conf/development-app-sentry.ini

```diff
@@ -0,0 +1,109 @@
+[pipeline:main]
```

Comment:
I guess you only used this to test Sentry in development? If so it can probably be omitted from these changes.